documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -190,6 +190,46 @@ spec:
     # ...
 ----
 
+For more information on configuring template properties, see xref:assembly-customizing-kubernetes-resources-str[].== Adding labels and annotations to Secrets
+
+By configuring the `jmxSecret` template property in the `Kafka` custom resource, you can add custom labels and annotations to the JMX authentication Secrets created by the Cluster Operator.
+Labels and annotations are useful for identifying objects and adding contextual information.
+You configure template properties in Strimzi custom resources.
+
+.Example template customization to add labels and annotations to Secrets
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    template:
+      jmxSecret:
+        metadata:
+          labels:
+            label1: value1
+            label2: value2
+          annotations:
+            annotation1: value1
+            annotation2: value2
+    # ...
+  zookeeper:
+    # ...
+    template:
+      jmxSecret:
+        metadata:
+          labels:
+            label1: value1
+            label2: value2
+          annotations:
+            annotation1: value1
+            annotation2: value2
+    # ...
+----
+
```

Comment:
Do we really want a dedicated documentation and example for this? Most of the other template is IMHO not covered to such detail. It increases the maintenance effort and I'm not sure it really adds value. Just think about it.